### PR TITLE
UAV Tech 5" Preset

### PR DESCRIPTION
**Pull request by spatzengr**
UAV Tech 5" Preset

PID Tab:
image

Filters Tab:
image

48K PWM Option:
image

### CODE REVIEW

File: presets/4.3/tune/UAV_tech_5in.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#$ TITLE: UAV Tech - 5in (~630g AUW)
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: OFFICIAL
```
Comment (limonspb) [inline presets/4.3/tune/UAV_tech_5in.txt:4]: all-new presets should first go to as "EXPERIMENTAL". And later on with some feedback can become community and then official

File: presets/4.3/tune/UAV_tech_5in.txt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#$ KEYWORDS: 5in, 5", freestyle
+#$ AUTHOR: UAV Tech (Mark Spatz)
+#$ DESCRIPTION: Mark is a Betaflight contributor, youtube content creator, and professional tuner (www.theuavtech.com)
```
Comment (limonspb) [inline presets/4.3/tune/UAV_tech_5in.txt:7]: could you please specify:
is that 4s, 6S or both? Can add 4S, 6S keywords too.
Is that for RPM filters ON or OFF? If this tune is for RPM-only, you can set dshot600/dshot300 and set bdir ON
In this case don't forget about
```
#$ INCLUDE_WARNING: misc/warnings/en/dshot.txt
#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
```

Comment (limonspb) [inline presets/4.3/tune/UAV_tech_5in.txt:7]: this is optional, but you can use:
`#$ PARSER: MARKED`
and then can at least insert links people can click on, for example:
`[www.theuavtech.com](https://www.theuavtech.com)`

**Review by limonspb — CHANGES_REQUESTED** [review]

**Review by limonspb — APPROVED** [review]

**Comment by ombracol** [comment]
image


When I load 5 inch  preset  it disables slider in gyro filter multiplier  and also turns off gyro lowpass 1 and 2. doesnt seem right to me. but maybe thats whats intended? , did i miss something? 
I loaded this on a speedbeef7v2 . I was able to duplicate on 2 seperate boards,  
cli shows gyro lowpass at 0

**Comment by spatzengr** [comment]
> When I load 5 inch preset it disables slider in gyro filter multiplier and also turns off gyro lowpass 1 and 2. doesnt seem right to me. but maybe thats whats intended? , did i miss something? I loaded this on a speedbeef7v2 . I was able to duplicate on 2 seperate boards, cli shows gyro lowpass at 0

This is by design as shown in the Filter image for the preset above (original post of submitted preset).  With 4 notches enabled, or RPM plus 2 notches, that should be adequate for gyro filtering.  HOWEVER, in your screenshot, it does seem like you changed the RPM and Dynamic Notch count to 3 and 1.

**Comment by ombracol** [comment]
> > When I load 5 inch preset it disables slider in gyro filter multiplier and also turns off gyro lowpass 1 and 2. doesnt seem right to me. but maybe thats whats intended? , did i miss something? I loaded this on a speedbeef7v2 . I was able to duplicate on 2 seperate boards, cli shows gyro lowpass at 0
> 
> @ombracol: This is by design as shown in the Filter image for the preset above (original post of submitted preset). With 4 notches enabled, or RPM plus 2 notches, that should be adequate for gyro filtering. HOWEVER, in your screenshot, it does seem like you changed the RPM and Dynamic Notch count to 3 and 1.

Thank you I guess I freaked out because I was getting Gyro overflow , but silly me forgot to remap my motors when I flashed 4.3  and didnt read above comments  :-D  Thank for the response great work !

**Comment by Bryanw55** [comment]
Tried this today on my rather porky (735-770g AUW without camera) I got a very high pitched whine/whistle coming from the motors on the hover test, so I went back to a different preset. I am using beta firmware Blheli_32_83 on the ESC that speedybee gave me to test, since it is one of the Speedybee V2s that had been problematic. It has been working fine on the BLheli beta and with Betaflight 4.3 stock and Supafly preset.   I'm set on 24kHz pwm frequency.

**Comment by Kevin590** [comment]
This tune works great on my 5" freestyle build with AUW 550g with Naked Gopro Hero 7 Black camera. Motors come down cool to the touch and it flies like it's on rails.

Build:

- TCMM x220hv frame
- JHEMCU GHF411 PRO 25A AIO FC w/Bluejay 24KHz
- JHEMCU ELRS EP24S RX 250Hz BF 4.3 preset
- DYS 2304 2300kv Race Motors
- HDZero Whoop VTX
- 4S 1300mah Battery

Options:

- RPM Filtering
- Dynamic Idle

AmericaQuad1

**Comment by bmxdougherty** [comment]
How much will weight effect the tune? My AUW is 589-602 depending on battery. But when I add a GP8 I am around 720. How fragile is the tune when it comes to weight?

**Comment by spatzengr** [comment]
> How much will weight effect the tune? My AUW is 589-602 depending on battery. But when I add a GP8 I am around 720. How fragile is the tune when it comes to weight?

Moment of Inertia (MoI) an prop trust differences are the impactors to each tune.  

If MoI goes up, but thrust does not, you will need more Damping and higher gains (Master slider).  

If MoI goes down or thrust goes up, less Damping and lower gains (Master slider)

**Comment by tlebrell** [comment]
I took about a 1.5 year break from flying and just got got back into the hobby. I used to run a drone shop so I have experience building and tuning, but I wasn't ready for Betaflight 4.3. I flashed up a buddy's quad a little while back and used the UAVTech 5" preset and the motors ran super hot, so we shelved it. I just got my build done today and I used your 5" quad profile again, and my motors got real hot real fast. It even shut down ESC a couple of times. Heck, my motors even sounded weird. I used to use your settings from your tuning page, and they never disappointed, so I'm not sure what's going on. I reverted back to stock settings, and other than crappy handling, it flies now. What do you think I might be doing wrong?

**Comment by spatzengr** [comment]
> What do you think I might be doing wrong?

Did you check your build and electrical noise?  If not, that's step #1 after any new build!  Don't assume it is good.  If you are having trouble with this preset, it is most likely a mechanical/electrical issue you need to sort.  Of course, I could be wrong (as proven through logging GYRO_SCALED  ;-) ).

Check out: https://youtu.be/n2-fj9EOheI

**Pull request merged**